tasks/patternpack.js

```diff
@@ -3,12 +3,12 @@ module.exports = function (grunt) {
 
   var log = require("../gruntLogHelper.js")(grunt);
   var fs = require("fs");
+  var path = require("path");
   var _ = require("lodash");
   _.defaultsDeep = require("merge-defaults"); // Add deep defaults capabilities to lodash
 
-  var npmPath = "./node_modules/";
-  var packageName = "patternpack";
-  var packagePath = npmPath + packageName;
+  var packagePath = "./" + path.relative(process.cwd(), path.dirname(__dirname)); // "./node_modules/patternpack"
```

Comment:
Made the `packagePath` and `packageName` values dynamically based upon the PatternPack package, so that they are no longer hardcoded.